r/R/field.R

```diff
@@ -54,11 +54,11 @@ Field <- R6Class("Field",
     }
   )
 )
-Field$create <- function(name, type, metadata) {
+Field$create <- function(name, type, metadata = NA, nullable = TRUE) {
```

Comment:
We don't need to provide a default argument to metadata 

```suggestion
Field$create <- function(name, type, metadata, nullable = TRUE) {
```